modules/user_manual/pages/files/federated_cloud_sharing.adoc

```diff
@@ -8,7 +8,7 @@ Federation Sharing allows you to mount file shares from remote ownCloud
 servers, in effect creating your own cloud of ownClouds. You can create
 direct share links with users on other ownCloud servers.
 
-IMPORTANT: Sharing to groups from federated ownCloud instances is not supported.
+IMPORTANT: https://doc.owncloud.org/server/admin_manual/configuration/files/federated_cloud_sharing_configuration.html[Federated Sharing] has to be configured by the administrator.
```

Comment:
```suggestion
IMPORTANT: xref:admin_manual:configuration/files/federated_cloud_sharing_configuration.adoc[Federated Sharing] has to be configured by the administrator.
```
DO NOT USE for references inside docs html links!!!
USE xref pointing to a .adoc file instead